client/lib/analytics/docs/page-views.md

```diff
@@ -0,0 +1,86 @@
+Analytics: Page Views
+=====================
+
+Page views should be used to record all page views (when the main content body completely changes). This will automatically record the page view to both Google Analytics and Tracks.
```

Comment:
This first couple of sentences would make more sense as:
 

> Page views should be recorded every time the main content body completely changes. These tools will automatically record stats to both Google Analytics and Tracks.

Reply:
What if we also noted both the refresh/click case?

> Page views should be recorded every time the main content body completely changes. This include both when the browser is refreshed, and when any time the browser url is updated via the browser history API...